src/_posts/hands-on-i.md

```diff
@@ -0,0 +1,143 @@
+
+I chose to implement one step at a time without realistic models in the beginning. Once I am able to recreate the complete process, I can think about adding different levels of detail to the demo.
+
+## Setting the scene up
+
+Let’s start looking at the code. The first thing I needed was a scene containing all the elements I wanted to work with: the air gun, the suction pad, the chip, the soldering around the chip, and the circuit board. Of course I needed a representation of my own hands and some environmental elements such as a sky, a floor and of course a table!
+
+This is the initial code for the scene:
+
+```html
+<a-scene>
+  <!-- World -->
+  <a-sky color="#333"></a-sky>
+  <a-plane rotation="-90 0 0"
+           width="15" height="15" color="#777">
+  </a-plane>
+  <!-- Workspace -->
+  <a-entity id="workspace" position="0 0.8 -0.8">
+    <a-box id="workbench" position="0 0 0"
+           width="2" depth="1" height="0.02" color="#97b7d5">
+    </a-box>
+    <a-box id="circuitboard" position="0 0.0115 0.4"
+           width="0.05" depth="0.05" height="0.003" color="#008000">
+      <a-box id="chip" position="0 0.002 0.005"
+             width="0.02" depth="0.02" height="0.001" color="#000">
+      </a-box>
+    </a-box>
+    <a-cylinder id="suction-pad"
+                position="-0.2 0.015 0.286" rotation="-90 0 0"
+                height="0.1" radius="0.005" color="red">
+    </a-cylinder>
+    <a-cylinder id="hot-air-gun"
+                position="0.3 0.02 0.286" rotation="-90 0 0"
+                height="0.15" radius="0.01" color="blue">
+    </a-cylinder>
+  </a-entity>
+  <!-- Operator -->
+  <a-camera position="0 0 0" user-height="1.3"></a-camera>
+  <a-entity id="left-hand" hand-controls="left"></a-entity>
+  <a-entity id="right-hand" hand-controls="right"></a-entity>
+</a-scene>
+```
+
+When working with A-Frame, all the scene objects are contained inside the `a-scene` element. A-Frame provides some special tags to achieve common tasks such as adding a sky dome or a camera into the virtual space and it also has primitives available such as boxes, planes, or cylinders.
+
+This is how I set up the environment with a sky dome, and a plane decorated with a couple of sad grey tones.
+
+```html
+<a-sky color="#333"></a-sky>
+<a-plane rotation="-90 0 0"
+         width="15" height="15" color="#777">
+</a-plane>
+```
+
+Usually, units in A-Frame are those of the [**international system**](https://en.wikipedia.org/wiki/International_System_of_Units) except for angles, which are expressed in sexagesimal degrees (0º to 360º). Thus, distances and point coordinates are expressed in meters and so the floor has a surface of _15m x 15m_, the workbench is _2cm_ thick and has a surface of _2m x 1m_, and the circuit board is _3mm thick_ and measures _5cm x 5cm_. Real measures are important for two reasons: they **improve the feeling of immersion** and provide **realistic physics**.
```

Comment:
I would just say `meters` rather than explaining international system or degrees. A-Frame doesn't use cm or mm either (or celcius, liters, etc). The only reason is because the WebVR API returns units in meters.